src/editors/data/services/cms/api.ts

```diff
@@ -116,17 +118,15 @@ export const apiMethods = {
     urls.blockStudioView({ studioEndpointUrl, blockId }),
   ),
   fetchImages: ({
+    blockId,
     learningContextId,
     studioEndpointUrl,
     pageNumber,
   }): Promise<{ data: AssetResponse & Pagination }> => {
     if (isLibraryKey(learningContextId)) {
-      // V2 content libraries don't support static assets yet:
-      return Promise.resolve({
-        data: {
-          assets: [], start: 0, end: 0, page: 0, pageSize: 50, totalCount: 0,
-        },
-      });
+      return get(
+        `${urls.libraryAssets({ studioEndpointUrl, blockId })}`,
+      );
```

Comment:
According to line 125, this function returns `Promise<{ data: AssetResponse & Pagination }>` but here you are returning totally different data (it has `files` instead of `assets`), and then transforming it in `thunkActions/requests.js` (`fetchImages`). Please make sure that the data being returned here matches the declared return type of the function, either by changing the return type or by transforming it within the function. Or, simply make a separate function to retrieve the library images instead of the course images.

(This would be obvious if `thunkActions/requests.js` was using TypeScript or had `// @ts-check` on its first line, but unfortunately that file has a ton of type errors and problems, so it's still JS at the moment.)